src/WikiService.Models.Dto/Requests/Article/CreateArticleRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using LT.DigitalOffice.WikiService.Models.Dto.Models;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.WikiService.Models.Dto.Requests.Article
+{
+  public record CreateArticleRequest
+  {
+    public string Name { get; set; }
+    public string Content { get; set; }
+    public bool IsActive { get; set; }
```

Comment:
remove. 
the user sets the IsActive for delayed publications, this service does not need delayed publications